_posts/2020-07-13-scipy-2020-maintainers-track.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: SciPy 2020 Maintainers Track Talk
+tagline: An update on Dask over the last 12 months
+author: Jacob Tomlinson (NVIDIA)
+tags: [SciPy, Community, Talk]
+theme: twitter
+---
+
+{% include JB/setup %}
+
+We recently enjoyed the 2020 SciPy conference from the comfort of our own homes this year. The 19th annual Scientific Computing with Python conference was a virtual conference this year due to the global pandemic. The annual SciPy Conference brought together over 1500 participants from industry, academia, and government to showcase their latest projects, learn from skilled users and developers, and collaborate on code development.
+
+This year we had the privilege of giving the first project update on the maintainers track.
```

Comment:
There were three days I think, of which we were the last day. Regardless, I think that this bit of information isn't critical.

Reply:
Ah fair enough!